### PR TITLE
VATRP-3619, VATRP-3596

### PR DESCRIPTION
**Pull request by samson-accontech**
Video mail indicator is added, with its functionality.

### CODE REVIEW

File: src/org/linphone/LinphoneActivity.java [hunk 1]
```diff
@@ -493,16 +488,16 @@ public void onClick(View v) {
 
 	private void videoMail() {
 		try {
-			LinphoneManager.getInstance().newOutgoingCall(mPrefs.getString(getString(R.string.pref_voice_mail_key), LinphoneManager.getLc().getDefaultProxyConfig().getAddress().asStringUriOnly()), getResources().getString(R.string.main_menu_videomail));
+			LinphoneManager.getInstance().newOutgoingCall(mPrefs.getString(getString(R.string.pref_voice_mail_key), ""), getResources().getString(R.string.main_menu_videomail));
```
Comment (zackm0571) [inline src/org/linphone/LinphoneActivity.java]: Revert this line. This will undo the default video mail functionality.

**Comment by zackm0571** [comment]
By the way, this implementation only increments the badge it doesn't use the value retrieved from the mail waiting payload.

**Comment by kennethdick** [comment]
Do other apps just increment also? Consistency is important

File: src/org/linphone/LinphoneActivity.java [hunk 2]
```diff
@@ -292,20 +294,20 @@ public void onClick(DialogInterface dialog, int which) {
...
-								reloadMwiCount();
-							}
+					try {
+						SharedPreferences prefs = getSharedPreferences(getPackageName(), MODE_PRIVATE);
+						String key = "Messages-Waiting: ";
+						String data = new String(content.getData());
+						data = data.substring(data.indexOf(key) + key.length());
+						int number = Integer.parseInt(data.substring(0, data.indexOf("\r\n")));
+
+						prefs.edit().putInt("mwi_count", number).commit();
```
Comment (ianblenke) [inline src/org/linphone/LinphoneActivity.java:304]: @zackm0571, this looks like the mwi_count is being parsed out of the Messages-Waiting: line in the body of the MWI message. This is correct behavior. It is not simply incremented anymore.

**Comment by kennethdick** [comment]
Add MWI notification on bottom with count from the MWI message

**Comment by ianblenke** [comment]
+1 LGTM

**Pull request merged**